zebra/rt_netlink.c

```diff
@@ -1021,6 +1021,9 @@ static int netlink_route_change_read_multicast(struct nlmsghdr *h,
 	char oif_list[256] = "\0";
 	vrf_id_t vrf;
 	int table;
+	void *src = NULL;
+	void *grp = NULL;
+	char anyaddr[16] = {0};
```

Comment:
The way this is handled here with `anyaddr` is… weird and relatively bad coding style. Can you find some better way?

Reply:
Fixed